sdks/python/README.md

```diff
@@ -0,0 +1,3 @@
+# Redactive Python SDK
+
+The Redactive Python SDK provides a robust and intuitive interface for interacting with the Redactive platform, enabling developers to seamlessly integrate powerful data redaction and anonymization capabilities into their Python applications.
```

Comment:
nit: I thought we're using Australian spelling? Anonymisation.

Reply:
It is generated lol

Reply:
It should be fine. Will update in future PR.